addon/models/osf-model.js

```diff
@@ -17,6 +17,7 @@ import HasManyQuery from 'ember-data-has-many-query';
 export default DS.Model.extend(HasManyQuery.ModelMixin, {
     links: DS.attr('links'),
     embeds: DS.attr('embed'),
+    errors: DS.attr(),
```

Comment:
This is kind of an awful one, because by default typing `node.parent` would mess up the template. Ember assumes the relationship corresponds to a request that will work.

Here's how I hacked around it in the project navbar, which has a "parent is private" indicator... I checked that the parent relationship existed:
https://github.com/CenterForOpenScience/ember-osf/blob/develop/addon/components/eosf-project-nav/component.js#L49

And then a computed property that returned false if the parent couldn't be successfully fetched...
https://github.com/CenterForOpenScience/ember-osf/blob/develop/addon/components/eosf-project-nav/component.js#L55

There might be a more elegant way to go about it, so alternate ideas very welcome.